compositCommon/AMDSCommon.pri

```diff
@@ -4,7 +4,6 @@
 #
 #-------------------------------------------------
 
-
 QT       += core network gui sql
```

Comment:
No. The reason we have `sql` is that we are using some Acquaman codes, such as AMPVControl, which requires `sql` package